.github/workflows/build_docs.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow generates the Doxygen documentation
+name: Doxygen Action
+
+# Controls when the action will run. There’s no way to specify that a workflow should be
+# triggered when a pull request is merged. However, because a merged pull request always
+# results in a push, we can use a push event to develop to accomplish our goal.
+on:
+  workflow_dispatch:
+    push:
+      branches:
+      - develop
+      pull_request:
```

Comment:
I would remove this, as we dont want to deploy the docs on each opened pull request:

```suggestion
```